Automattic-Tracks-iOS/Event Logging/EventLoggingUploadManager.swift

```diff
@@ -61,4 +57,13 @@ class EventLoggingUploadManager {
             }
         }
     }
+
+    func createRequest(url: URL, uuid: String, authenticationToken: String) -> URLRequest {
+        var request = URLRequest(url: url)
+        request.addValue(uuid, forHTTPHeaderField: "log-uuid")
```

Comment:
By moving these values here, I don't anticipate we need them as constants anymore (if they were to change, the method wouldn't work anymore, and because the API is finalized, this should only change for a rev-bumped endpoint)